test/e2e/common/setup-kubestellar.sh

```diff
@@ -198,3 +198,4 @@ then
     echo "Failed to see two clusters."
     exit 1
 fi
+
```

Comment:
We do not need a blank line at the end of the file. In the current version of this PR, this file ends with two newlines. You only need one, to terminate the last line. Following is what I see now.

```console
mspreitz@mjs13 kubestellar % od -A x -t a -t x1 test/e2e/common/setup-kubestellar.sh | tail -3
0001d90   sp  sp  sp   e   x   i   t  sp   1  nl   f   i  nl  nl        
           20  20  20  65  78  69  74  20  31  0a  66  69  0a  0a        
0001d9e
```

Same for the other file modified in this PR.